
commit 291b54073509f7ed9d6d6ba34f5866e0f106186c
Author: Jeff Law <jeffreyalaw@gmail.com>
Date:   Tue Feb 4 13:47:51 2025 -0700

    Revert "nds32: use is_whitespace()"
    
    This reverts commit 1ac26e9f7ac250cd63f766eae0facea3485b16e4.

diff --git a/gas/config/tc-nds32.c b/gas/config/tc-nds32.c
index 2e0b8e59099..3de27fdcea9 100644
--- a/gas/config/tc-nds32.c
+++ b/gas/config/tc-nds32.c
@@ -3452,7 +3452,7 @@ nds32_lookup_pseudo_opcode (const char *str)
 
   for (i = 0; i < maxlen; i++)
     {
-      if (is_whitespace (op[i] = str[i]))
+      if (ISSPACE (op[i] = str[i]))
 	break;
     }
   op[i] = '\0';
